extensions/objects/models/objects_test.py

```diff
@@ -26,25 +26,27 @@
 class ObjectNormalizationUnitTests(test_utils.GenericTestBase):
     """Tests normalization of typed objects."""
 
-    def check_normalization(self, cls, mappings, invalid_items):
+    def check_normalization(self, object_class, mappings, invalid_items):
```

Comment:
Hmm..I am not sure about making changes here. Shouldn't we incorporate such cases where `self` and `cls` both are present as arguments, that `self` should come first and then `cls`?

Reply:
Actually @seanlip suggested these changes, in the [#5844(comment)](https://github.com/oppia/oppia/pull/5844#discussion_r232437005) . As there were only two cases present . Well we can add such case also where ```self``` and ```cls``` both are present  and self should come first and then cls. What you suggest?

Reply:
Oh, then it's fine here. Thanks for the link to the comment!